paper/content/chapter2/chapter2_1.tex

```diff
@@ -1,9 +1,156 @@
 
 
-\subsubsection{Weitere Evaluierungsfunktionen}
+\subsubsection{Einfache Bewertungsfunktion}
+
+Das Schachspiel setzt sich aus zwei wichtigen Faktoren zusammen: Einerseits ist abgesehen von der Tatsache, welcher Spieler den ersten Zug macht, keine Zufallskomponete enthalten, andererseits handelt es sich um ein Spiel mit \gls{PI}.
+Diese zwei Faktoren führen dazu, dass bei jeder Schachbrettposition eine der folgenden drei Aussagen gilt [\cite{Shannon1950}]:
+
+\begin{enumerate}
+    \item Es handelt sich um eine gewonnene Position für Weiß. Somit kann Weiß einen Sieg forcieren, wobei Schwarz verteidigen.
+    \item Es handelt sich um eine gewonnene Position für Schwarz. Somit kann Schwarz einen Sieg forcieren, wobei Weiß spielt.
+    \item Es handelt sich um eine unentschiedene Position für beide Parteien. Somit kann es nur ein Unentschieden am Ende geben, falls beide Parteien dies forcieren und keine Fehler machen.
+\end{enumerate}
+
+\noindent Bei einigen Spielen (so auch beim Schach) lässt sich aus den genannten zwei Faktoren und den daraus resultierenden drei Aussagen eine \textit{einfache Bewertungsfunktion} \(\displaystyle f(P)\) ableiten, wobei \(\displaystyle P\) die Schachbrettposition bezeichnet.
+Der Rückgabewert der Funktion ist die Kategorie, in die die jeweilige Position gehört: Sieg (+1), Unentschieden (0), Niederlage (-1).
+Zum Zeitpunkt des Zuges des Schachcomputers werden die Werte \(\displaystyle f(P)\) für alle Positionen nach möglichen Halbzügen berechnet.
+Der Zug mit dem maximalen Wert wird am Ende ausgeführt [\cite{Shannon1950}].
+
+Im Zuge dieser Arbeit wird die einfache Bewertungsfunktion nach Tomasz Michniewski verwendet, die ursprünglich in der \textit{Polish chess programming discussion list (progszach)} veröffentlicht wurde und im chessprogramming.org Wiki beschrieben wird.
+Die Gewichtungen der Bewertungsfunktion wurden so gewählt, dass ein Mangel an Schachkenntnissen ausgeglichen werden kann und nicht, um durch Schachkenntnisse ergänzt zu werden.
+Da die Gewichtungen von Tomasz Michniewski und nicht von den Autoren dieser Arbeit festgelegt wurden, erfolgt an dieser Stelle keine Erläuterung zur Entstehung der Gewichtungswerte [\cite{Wiki2018}]. Die von Tomasz Michniewski beschriebene einfache Bewertungsfunktion wird in zwei Teilen dargestellt: Einfache Figurenwerte (engl. \textit{simple piece values}) und Figuren-Quadrat Tabellen (engl. \textit{piece-square tables}).
+
+Mit der Festlegung von Werten je Figur werden vier verschiedene Ziele erreicht:
+
+\begin{enumerate}
+    \item Vermeidung des Austauschs einer kleinen Figur gegen drei Bauern
+    \item Dem Computer signalisieren, dass das halten des Läuferpaars vorteilhaft ist
+    \item Vermeidung des Austauschs von zwei kleinen Figuren gegen einen Turm und einen Bauern
+    \item Verbleiben bei der menschlichen Schacherfahrung
+\end{enumerate}
+
```

Comment:
Letzten Punkt in die Beschreibung mit rein nehmen. Also sowas wie: drei verschiedene ziele um bei menschlicher Schacherfahrung zu verbleiben..

Reply:
Er nennt in seiner Mail-Korrespondenz das aber als einen der vier Punkte, die er so erreichen will (fast 1-zu-1 übersetzt).

Reply:
Meinst du mit er Strohmann? Weil die anderen Punkte sind ja konkrete Gleichungen, welche alle den 4. Punkt als Ziel haben. Finde der passt nicht so schön dazu, weil ja auch keine Gleichung aus ihm resultiert und er an sich ziemlich wage ist. Aber wenn er es explizit so haben wollte dann lass es natürlich so.

Reply:
Und ja klar möchte er den erreichen, aber das ist keine konkrete Maßnahme sondern nur Teil des "bigger pictures", verstehst du was ich meine?